dev-util/github-cli/github-cli-0.9.0-r1.ebuild

```diff
@@ -294,4 +294,10 @@ src_compile() {
 src_install() {
 	dobin bin/gh
 	dodoc README.md
+
+	make manpages
+	doman share/man/man1/gh*.1
+
+	bin/gh completion -s bash > gh
```

Comment:
This command needs || die.

Also, this will fail when cross-compiling for another CPU, but I'm not sure if there's a nice way to detect that.

Reply:
The cross-compile issue was the only outstanding thing here, I fixed the rest.